helmfile/values/kured.yaml.gotmpl

```diff
@@ -35,3 +35,6 @@ slack:
   channel: {{ .Values.kured.notification.slack.channel }}
   botToken: {{ .Values.kured.slack.botToken }}
 {{- end }}
+
+image:
+  repository: ghcr.io/kubereboot/kured
```

Comment:
Any reason for the change of repository?

Reply:
https://github.com/kubereboot/charts/tree/main/charts/kured#from-3x-to-4x

Reply:
but according the the helm chart values that repo is set by default
https://github.com/kubereboot/charts/blob/kured-4.4.1/charts/kured/values.yaml#L1-L2